mpmp/src/controller/Controller.java

```diff
@@ -99,27 +134,34 @@ public void actionPerformed(ActionEvent e) {
 			}
 		});
 
-		frame.addSurrenderListener(new ActionListener() {
+		frame.addPayPrisonListener(new ActionListener() {
 			@Override
 			public void actionPerformed(ActionEvent e) {
-				conn.send("ragequit");
+				//conn.send("pay-prison");
```

Comment:
You might want to read the protocol spec to know that the packet is `unjail money` :wink: